Sources/Onboarding/OnboardingVPNDetailsView.swift

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Shared
+import BraveUI
+import BraveShared
+
+public struct OnboardingVPNDetailsView: View {
+  public var learnMore: (() -> Void)?
+  
+  private let descriptionItems = [Strings.VPN.checkboxBlockAds,
+                          Strings.VPN.popupCheckmarkSecureConnections,
+                          Strings.VPN.checkboxFast,
+                          Strings.VPN.popupCheckmark247Support]
+
+  public init() {}
+
+  public var body: some View {
+    VStack(spacing: 16) {
+      VStack(spacing: 12) {
+        Image("vpn_popup_shield", bundle: .module)
+          .padding(.bottom)
+          .accessibilityHidden(true)
+        Text(Strings.VPN.vpnName)
+          .font(.title.weight(.heavy))
+          .foregroundColor(Color(.bravePrimary))
+          .multilineTextAlignment(.center)
+        HStack(spacing: 8) {
+          Text(Strings.VPN.poweredBy)
+            .font(.footnote)
+            .foregroundColor(Color(.bravePrimary))
+            .multilineTextAlignment(.center)
+          Image(sharedName: "vpn_brand")
+            .renderingMode(.template)
+            .foregroundColor(.black)
+        }
+        .padding(.vertical, 12)
+      }
+      .frame(maxWidth: .infinity)
+      VStack(alignment: .leading, spacing: 8) {
+        ForEach(descriptionItems, id: \.self) { itemDescription in
+          HStack(spacing: 8) {
+            Image(sharedName: "vpn_checkmark_popup")
+              .renderingMode(.template)
+              .foregroundColor(.black)
```

Comment:
You're using `braveBackground` for the main VStack which means you'll get a dark-navy in dark mode which would maybe make this hard to see? Possibly need to use `bravePrimary` here as well

Reply:
You are totally right. I used it black because it was that way in previous view.

Changed it to bravePrimary. I checked both dark and light mode.
Fixed at https://github.com/brave/brave-ios/pull/6355/commits/ac121352cddc8af9dc4912b0d17abc72b4755051